model/labels.go

```diff
@@ -106,17 +106,17 @@ type LabelName string
 // IsValid returns true iff the name matches the pattern of LabelNameRE when
 // NameValidationScheme is set to LegacyValidation, or valid UTF-8 if
 // NameValidationScheme is set to UTF8Validation.
-func (ln LabelName) IsValid() bool {
+func (ln LabelName) IsValid(scheme ValidationScheme) bool {
 	if len(ln) == 0 {
 		return false
 	}
-	switch NameValidationScheme {
+	switch scheme {
 	case LegacyValidation:
 		return ln.IsValidLegacy()
 	case UTF8Validation:
 		return utf8.ValidString(string(ln))
 	default:
-		panic(fmt.Sprintf("Invalid name validation scheme requested: %d", NameValidationScheme))
+		panic(fmt.Sprintf("Invalid name validation scheme requested: %d", scheme))
```

Comment:
Please use the string representation:
```suggestion
		panic(fmt.Sprintf("Invalid name validation scheme requested: %s", scheme))
```